docs/getting-started/05-add-redis-service.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Add the Redis service
+type: Getting Started
+---
+
+Provision the external [Redis](https://redis.io/) service that will replace your microservice in-memory storage. To do so, you will use a sample addon configuration linking to an example in the GitHub repository.
+
+## Reference
+
+This guide demonstrates how [Service Catalog](/components/service-catalog/) works in Kyma. It enables easy access to services ([ServiceClass CRs](https://svc-cat.io/docs/resources/#service-classes)) managed by [Service Brokers](/components/service-catalog/#overview-service-brokers) registered in Kyma. You can consume these services by creating their instances and binding them to your microservices and Functions.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-redis-service">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Provision an [AddonsConfiguration CR](/components/helm-broker/#custom-resource-addons-configuration) with the Redis service:
+
+   ```yaml
+   cat <<EOF | kubectl apply -f  -
+   apiVersion: addons.kyma-project.io/v1alpha1
+   kind: AddonsConfiguration
+   metadata:
+     name: redis-addon
+     namespace: orders-service
+   spec:
+     repositories:
+       - url: https://github.com/kyma-project/addons/releases/download/0.12.0/index-testing.yaml
+   EOF
+   ```
+
+2. Check that the AddonsConfiguration CR was created. Its phase should state `Ready`:
```

Comment:
```suggestion
2. Check that the AddonsConfiguration CR was created. This is indicated by the status phase `Ready`:
```
As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology), we're not supposed to use `should`. What if it doesn't?

Another way to reword it would be: ```Check that the AddonsConfiguration CR was created. It's created when the status phase states `Ready`.```, but I like the original suggestion more, plus it's consistent with the suggestion I made to the next doc in line 35.

Reply:
I applied the 1st suggestion, just changes the order into:
```This is indicated by the `Ready` status phase```